tests/integration/suite/actors/healthz/endpoint/noappentities.go

```diff
@@ -70,7 +70,7 @@ func (n *noappentities) Setup(t *testing.T) []framework.Option {
 
 	srvNoHealthz := prochttp.New(t,
 		prochttp.WithHandlerFunc("/dapr/config", func(w http.ResponseWriter, r *http.Request) {
-			w.Write([]byte(`{}`))
+			w.Write([]byte(`{"entities": []}`))
```

Comment:
How does this fix the issue? If `entities` is unset and it's causing issues, maybe this is a bug in Dapr?

Reply:
I'm going to open up a follow-up test PR to ensure that an empty object can be parsed by `/dapr/config` successfully- this may be a bug in the config client.

Reply:
Why is this a bug on MacOs only?

Reply:
> Why is this a bug on MacOs only?

My question exactly